### ISSUE TITLE
Please add an option to get not only full-screen screenshot, but also window-only

### ISSUE THREAD

**Issue by ssten** [FreeRDP/Remmina#1324]
For now I use *net-misc/remmina (1.2.0_rc21@10/16/17)* in Gentoo GNU/Linux *amd64* arch.

Screenshot button is a useful feature.
But it is incomplete.

Rather often I need not a full-screen screenshot, but just an active window.
At least for *rdp* connections, windows remote host has needed feature (*Alt*-*Prt Scrn* shortcut).

It would be fine, if I'll be able to get the same windon-only screenshot directly from *remmina*.

**Comment by giox069** [comment]
I don't think we can have the coordinates of a window via RDP or VNC protocol: they only send us graphical commands or bitmap, so remmina cannot know where windows are placed.
But we could implement a rectangular area capture in the future.
Have you tried to enable keyboard capture (on the remmina toolbar) and just press Alt+PrtScrn when the mouse cursor is inside the connection windo, and then paste the image locally?
It works, but I think it currently could not work under Wayland (keyboard capture is not possible). And if the image is not pasted immedialtely, please retry the paste operation after some seconds.

**Comment by ssten** [comment]
Does remmina override freerdp's default to disable clipboard?
Getting image from window's clipboard is a solution.
But for now on initila test I've find pre-set «Send Ctrl-Alt-Del» inoperable. How can I debug it?

**Comment by antenore** [comment]
WARNING:
The Remmina project is moving to gitlab.com, in order to keep receiving notification about this issue,
you should have a gitlab account and link it to github.
The migration will happen in the next days.
Thanks for your support and understanding.

cc @giox069 @ssten
